proposals/simd/SIMD.md

```diff
@@ -27,17 +28,34 @@ encoded as individual bytes in the binary encoding. Many have a limited valid
 range, and it is a validation error if the immediate operands are out of range.
 
 * `ImmByte`: A single unconstrained byte (0-255).
-* `LaneIdx2`: A byte with values in the range 0–1 identifying a lane.
-* `LaneIdx4`: A byte with values in the range 0–3 identifying a lane.
-* `LaneIdx8`: A byte with values in the range 0–7 identifying a lane.
-* `LaneIdx16`: A byte with values in the range 0–15 identifying a lane.
-* `LaneIdx32`: A byte with values in the range 0–31 identifying a lane.
-
-## Interpreting the SIMD value type
-
-The single `v128` SIMD type can represent packed data in multiple ways.
-Instructions specify how the bits should be interpreted through a hierarchy of
-*interpretations*.
+* `ImmLaneIdx2`: A byte with values in the range 0–1 identifying a lane.
+* `ImmLaneIdx4`: A byte with values in the range 0–3 identifying a lane.
+* `ImmLaneIdx8`: A byte with values in the range 0–7 identifying a lane.
+* `ImmLaneIdx16`: A byte with values in the range 0–15 identifying a lane.
+* `ImmLaneIdx32`: A byte with values in the range 0–31 identifying a lane.
+
+## Operations on the SIMD value type
+
+The _single_ `v128` SIMD type can be used to represent different types of packed
+data, e.g., it can represent four 32-bit floating point values, 8 16-bit signed
+or unsigned integer values, etc.
+
+The instructions introduced in this specification are named according to the
+following schema: `{interpretation}.{operation}`. Where the `{interpretation}`
+prefix denotes how the bytes of the `v128` type are interpreted by the `{operation}`. 
+
+For example, the instructions `f32x4.extract_lane` and `i64x2.extract_lane`
+perform the same semantic operation: extracting the scalar value of a vector
+lane. However, the `f32x4.extract_lane` instruction returns a 32-bit wide
+floating point value, while the `i64x2.extract_lane` instruction returns a
+64-bit wide floating point value.
+
+The interpretations form a hierarchy with successive refinements, where `v128`
+is at the top of the hierarchy and interprets the `v128` vector type as a bag of
+bits. The `v32x4` refines `v128` (`v32x4 : v128`) and interprets the vector as 4
+32-bit wide lane, while the `i32x4` and `f32x4` both refine `v32x4` and
+interpret the content of the vector lanes as being of integer or floating-point
+types.
```

Comment:
I'm wondering if hierarchy is the right word here? To me, these are just different interpretations of v128 values.

Reply:
It did feel off to me as well, but that's the word the text was using :/ 

I would just write something like:

> The `v128` vector type interpretation interprets the vector as a bag of bits. The `v{lane_width}x{n}` interpretation (e.g. `v32x4`) interpret the vector as `n` lanes of `lane_width` bits. The `{t}{lane_width}x{n}` interpretations (e.g. `i32x4` or `f32x4`) interpret the vector as `n` lanes of type `{t}{lane_width}`.

Reply:
Thanks for clarifying, using the text in your comment sgtm.